pkgs/development/python-modules/pylibftdi/default.nix

```diff
@@ -0,0 +1,50 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, httpserver
+, libftdi1
+, libusb1
+}:
+
+buildPythonPackage rec {
+  pname = "pylibftdi";
+  version = "0.17.0";
+  disabled = pythonOlder "3"; # compatibility with httpserver
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "132c44rs0f2zm22hnhg93p47aaw6llpfzavc64dcqrn8719mlqfw";
+  };
+
+  propagatedBuildInputs = [
+    httpserver
+    libftdi1
+    libusb1
+  ];
+
+  # TODO Several modules have been renamed for python3. ReadTimeout
+  # doesn't appear to be part of pylibftdi at all... These changes
+  # should be upstreamed.
+  patches = [
+    ./make-python3-compatible.patch
+    ./fix-search-paths.patch
```

Comment:
This patch is pointless as you replace "find_library('libusb1')" again in the postPatch phase. Please remove.